### ISSUE TITLE
Add aria-hidden attribute to icons by default

### ISSUE THREAD

**Issue by lukonik** [ng-icons/ng-icons#172]
Most of the icons we use in the web apps don't have any contextual meaning for Screen Readers, so I think all the icons should have aria-hidden attribute by default. I would say global configuration is not needed to toggle this behavior because aria-hidden is so common for the icons

Angular material icon does exactly that
 https://material.angular.io/components/icon/overview#accessibility

Great library btw!

**Comment by lukonik** [comment]
also it needs role="img" for accessibility
https://developer.mozilla.org/en-US/docs/Web/Accessibility/ARIA/Reference/Roles/img_role#svg_and_roleimg

**Comment by lukonik** [comment]
I can create a PR for that ( probably on weekends ) if the issue is acceptable :)

**Comment by ashley-hunter** [comment]
Yes, this is something I have been deliberating about for a while. Originally I erred on the side of caution leaving it entirely to the user to define.

My hesitation was that if someone wanted to add an aria label (or similar attribute) they would have to know manually set aria-hidden to false, otherwise it wouldn't be read by a screen reader which may not be obvious.

That being said, I'm not against adding it into the component, and you are correct, in the vast majority of cases having aria-hidden set is a good default. And yes adding the img role to the host element too would also be a good move.

If you'd like to raise a PR for this at your convenience that would be great! Thank you. I'll perhaps include this as part of a major version bump as it could technically be considered a breaking change, but I'm ok with that.

**Comment by lukonik** [comment]
Yes good point, but I think most of the DEVs don't use accessibility attributes too much ;). So it will be nice to give them correct accessibility by default. We can also add the behavior to the documentation later.
Sounds great, thank you Ashley, then I'll work on it on weekends; I will keep you posted 👍

**Comment by ashley-hunter** [comment]
I agree 😊 thank you! Any issues just let me know! Thanks for the useful feedback!
